index.html

```diff
@@ -47,5 +47,332 @@ <h1 class="headline-1-h1">I’m Mariane
+                    </button>
+                </div>
+                <div class="works-2-div2-4">
+                    <button class="works-2-div2-4-btn" type="button">
+                        <span class="works-2-div2-4-btn-sp">See Project</span>
+                    </button>
+                </div>
+            </div>
+        </section>
+        <!--works3-->
+        <section class="works-3">
+            <div class="works-3-div1">
+                <img class="works-3-div1-img" src="images/Snapshoot Portfolio3.svg" alt="">
+            </div>
+            <div class="works-3-div2">
+                <div class="works-3-div2-1">
+                    <div class="works-3-div2-1-a">
+                        <p class="works-3-div2-1-a-p">Tonic</p>
+                    </div>
+                    <div class="works-3-div2-1-b">
+                        <ul class="works-3-div2-1-b-ul">
+                            <li class="works-3-div2-1-b-ul-li-1">
+                                <span class="works-3-div2-1-b-ul-li-1-sp">CANOPY</span>
+                            </li>
+                            <li class="works-3-div2-1-b-ul-li-2">
+                                <img class="works-3-div2-1-b-ul-li-2-img" alt="" src="images/Counter.svg">
+                            </li>
+                            <li class="works-3-div2-1-b-ul-li-3">
+                                <span class="works-3-div2-1-b-ul-li-3-sp">Back End Dev</span>
+                            </li>
+                            <li class="works-3-div2-1-b-ul-li-4">
+                                <img class="works-3-div2-1-b-ul-li-4-img" alt="" src="images/Counter.svg">
+                            </li>
+                            <li class="works-3-div2-1-b-ul-li-5">
+                                <span class="works-3-div2-1-b-ul-li-5-sp">2015</span>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="works-3-div2-2">
+                    <p  class="works-3-div2-2-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="works-3-div2-3">
+                    <button class="works-3-div2-3-btn1" type="button">
+                        <span class="works-3-div2-3-btn1-sp1">html</span>
+                    </button>
+                    <button class="works-3-div2-3-btn2" type="button">
+                        <span class="works-3-div2-3-btn2-sp2">css</span>
+                    </button>
+                    <button class="works-3-div2-3-btn3" type="button">
+                        <span class="works-3-div2-3-btn3-sp3">javaScript</span>
+                    </button>
+                </div>
+                <div class="works-3-div2-4">
+                    <button class="works-3-div2-4-btn" type="button">
+                        <span class="works-3-div2-4-btn-sp">See Project</span>
```

Comment:
- [ ] as part of the requirements kindly add the button interactions style  such as Active, Hover and Pressed to match the figma design , kindly fix it for all of your buttons

![image](https://user-images.githubusercontent.com/89282221/219620942-8cd56537-65b3-4ede-984a-c23a356ec53b.png)